internal/vrf/vrf.go

```diff
@@ -0,0 +1,52 @@
+package vrf
+
+import (
+	metal "github.com/equinix-labs/metal-go/metal/v1"
+	"github.com/equinix/metal-cli/internal/outputs"
+	"github.com/spf13/cobra"
+)
+
+type Client struct {
+	Servicer Servicer
+	Service  metal.VRFsApiService
+	Out      outputs.Outputer
+}
+
+func (c *Client) NewCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     `vrf`,
+		Aliases: []string{"vrf"},
+		Short:   "VRF operations : create, TODO: make other commands.",
```

Comment:
```suggestion
		Short:   "VRF operations : create, TODO: make other commands.",
```

TODO should be create, delete, retrieve